config/default/archives.yaml

```diff
@@ -0,0 +1,58 @@
+ingress:
+  enabled: true
+  annotations: 
+    "cert-manager.io/cluster-issuer": "letsencrypt-prod"
+    "kubernetes.io/ingress.class": "public-ingress"
+    "nginx.ingress.kubernetes.io/proxy-body-size": "500m"
+  hosts:
+    - host: archives.jenkins-ci.org
+      paths:
+        - "/"
+    - host: archives.jenkins.io
+      paths:
+        - "/"
+  tls:
+    - secretName: archivesmirror-tls
+      hosts:
+        - archives.jenkins-ci.org
+        - archives.jenkins.io
+
+replicaCount: 3
+
+synchronize:
+  enabled: false
+
+persistent:
+  volumeClaim:
+    name: archives-binary
+    enabled: true
+    spec:
+      accessModes:
+        - ReadWriteMany
```

Comment:
does it need to write? isn't it just read?

Reply:
You are right it needs to be read-only

Reply:
Fixed